content/faq-glossary-links/02-glossary.en.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Glossary
+weight: 1
+---
+
+## A
+
+## B
+
+### `branch`
+
+A line of development that stems from another branch, typically `main`/`master` or sometimes `dev`.
+
+## C
+
+### `checkout`
+
+Switching between working branches.
+
+### `cherry-picking`
+
+Choosing a subset of changes from other commits, typically on different branches.
+
+### `clone`
+
+Make a copy of a repository from a remote `origin` to your local computer.
```

Comment:
Calling the default remote 'origin' is just a convention.  Better to remove the word 'origin' from this definition and link to a definition in this glossary of what a remote is?